packages/@okta/vuepress-site/docs/guides/social-login/main/github/appidpinokta.md

```diff
@@ -0,0 +1,5 @@
+* **Name**: Enter a name for the Identity Provider in Okta.
+* **Client ID**: Paste the generated client ID from your <StackSnippet snippet="idp" inline /> application.
+* **Client Secret**: Paste the generated client secret from your <StackSnippet snippet="idp" inline /> application.
+* **Scopes**: Leave the defaults for a simple sign-in flow. You can also add scopes. See [Scopes for OAuth Apps](https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps)
+.
```

Comment:
```suggestion

```

Reply:
The period is for the previous line (sorry, bad formatting).